src/api/login/login.ts

```diff
@@ -52,9 +52,9 @@ router.post(
       password: req.body.password,
     }
     const user = await LoginModel.verify(data)
-    jwt.sign({ user: { user } }, 'secretKeyHere', { expiresIn: '30s' }, (err, token) => {
+    jwt.sign({ user: user }, 'secretKeyHere', { expiresIn: '30s' }, (err, token) => {
```

Comment:
havent we concluded that `{ user: user }` is the same as `{ user }`?
and `{ token: token }` as `{ token }`?